src/app/models/Project.ts

```diff
@@ -107,6 +101,13 @@ export class Project extends AbstractModel<IProject> implements IProject {
     };
   }
 
+  /**
+   * Returns true if user has the permissions to edit this model
+   */
+  public get canEdit(): boolean {
+    return [AccessLevel.owner, AccessLevel.writer].includes(this.accessLevel);
+  }
+
```

Comment:
did we set up a faked capabilities API? If so this method should fit into it.

Reply:
I remember some work being done on it, but I'm unsure where and if it resulted in something we can use. I believe accessLevel was what we ended up with